exercises/phone-number/test/phone_number_test.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:test/test.dart';
+import 'package:phone_number/phone_number.dart';
+
+final phoneNumber = new PhoneNumber();
+
+void main() {
+  group("PhoneNumber: Accept only numbers - ", numbersOnlyTest);
+  group("PhoneNumber: Number length - ", numberLenghtTest);
+  group("PhoneNumber: Cleanup special characters - ", cleanUpTest);
+  group("PhoneNumber: Area/Exchange Code - ", codeRangeTest);
+  group("PhoneNumber: Edge cases - ", edgeCases);
+}
+
+cleanUpTest() {
+  test("cleans the number", () {
+    expect(phoneNumber.cleanNumber("(223) 456-7890"), equals("2234567890"));
+  });
```

Comment:
Correct, the length of a number is 10 excluding the country code. And the edge case does have a valid number in it but because other special characters are also a part of the string it should be invalid.